webapp/app/forms/flows/step_chooser.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, title, steps, endpoint, form_storage, overview_step=None):
         self.overview_step = overview_step
 
     def _is_step_name_valid(self, step_name):
-        return step_name == "start" or step_name in self.steps
+        return step_name == "start" or step_name == "start_next" or step_name in self.steps
```

Comment:
Maybe we could rename "start_next" to "first_input_step" to make the idea more explicit?